Nos/Models/CoreData/NosNotification+CoreDataClass.swift

```diff
@@ -44,12 +44,14 @@ public class NosNotification: NSManagedObject {
         return fetchRequest
     }
     
+    // TODO: user is unused; is this a bug?
```

Comment:
@mplorentz I'd love your thoughts on this and the one below.

Reply:
yeah, we should be filtering notifications by `user` here or below. Otherwise you will see the wrong unread count after logging out and back in. But we have https://github.com/planetary-social/nos/issues/445 filed for that so if you want to get rid of the parameters for now that's fine.

Reply:
deleted the unused user params